ui/components/app/alert-system/confirm-alert-modal/confirm-alert-modal.tsx

```diff
@@ -40,16 +41,20 @@ function ConfirmButtons({
   onSubmit,
   isConfirmed,
 }: {
-  onCancel: () => void;
+  onCancel: ({ location }: { location: MetaMetricsEventLocation }) => void;
```

Comment:
Minor, but as we define this a few times, is it worth an `OnCancelHandler` type exported from `footer.tsx`?

Reply:
done